robozonky-strategy-natural/src/main/antlr4/imports/Tokens.g4

```diff
@@ -19,11 +19,6 @@ targetPortfolioSizeExpression returns [long result] :
     {$result = $maximumInvestmentInCzk.result;}
 ;
 
-targetBalanceExpression returns [long result] :
```

Comment:
FYI @jhrcek This is going away in 5.4.0, on account of Zonky implementing the PSD2 regulation. Also, it is not the only change in 5.4 - PSD2 will force us to make many more changes.

Reply:
Do you have more details about the planned changes? Can you share them or are they available somewhere? PM me on social media if you can share more :-)

Reply:
Sending PM.